src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -155,33 +153,30 @@ export class HostsComponent extends ListWithDetails implements OnInit {
         return { enabled: true, name: label };
       });
       this.modalService.show(FormModalComponent, {
-        titleText: this.i18n('Edit Host: {{hostname}}', host),
+        titleText: $localize`Edit Host: ${host}`,
```

Comment:
```suggestion
        titleText: $localize`Edit Host: ${host.hostname}`,
```

Reply:
good catch. I will have to double check if this isn't happening on other places...